src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -428,32 +435,37 @@ function ChildReconciler(shouldClone, shouldTrackSideEffects) {
           const created = createFiberFromElement(newChild, priority);
           created.ref = coerceRef(null, newChild);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_COROUTINE_TYPE: {
           const created = createFiberFromCoroutine(newChild, priority);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_YIELD_TYPE: {
           const created = createFiberFromYield(newChild, priority);
           created.type = newChild.value;
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
 
         case REACT_PORTAL_TYPE: {
           const created = createFiberFromPortal(newChild, priority);
           created.return = returnFiber;
+          created.contextTag = returnFiber.contextTag;
           return created;
         }
       }
 
       if (isArray(newChild) || getIteratorFn(newChild)) {
         const created = createFiberFromFragment(newChild, priority);
         created.return = returnFiber;
+        created.contextTag = returnFiber.contextTag;
```

Comment:
If we do that then we might want to change `unstable_asyncUpdates` to a static property instead of an instance property, so that it's available during reconciliation.